lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -48,18 +58,25 @@ class InspectorIssues extends Gatherer {
     driver.off('Audits.issueAdded', this._onIssueAdded);
     await driver.sendCommand('Audits.disable');
     const artifact = {
-      /** @type {Array<LH.Crdp.Audits.MixedContentIssueDetails>} */
+      blockedByResponse: [],
       mixedContent: [],
     };
 
     for (const issue of this._issues) {
-      if (issue.details.mixedContentIssueDetails) {
-        const issueDetails = issue.details.mixedContentIssueDetails;
-        const issueReqId = issueDetails.request && issueDetails.request.requestId;
-        if (issueReqId &&
-          networkRecords.find(req => req.requestId === issueReqId)) {
-          artifact.mixedContent.push(issue.details.mixedContentIssueDetails);
-        }
+      const details = getDetails(issue);
+      if (!details) continue;
+
+      const issueRequestId = details.request && details.request.requestId;
+      const issueRequest = networkRecords.find(req => req.requestId === issueRequestId);
```

Comment:
regardless of if we revert the artifact to be like the protocol object, it seems like this should be moved to the individual audits rather than apply to all the Issues